docs/Course Wiki/ECE Course Offerings/ECE330.md

```diff
@@ -0,0 +1,60 @@
+# ECE330
+
+ECE330 is one of the 3 of 5 advanced core electives required for electrical engineering majors and is one of the 1 of 6 electrical engineering foundations electives required for computer engineering majors.
```

Comment:
Format like this:
ECE330 (Course name here) is a 3-credit-hour course that satisfies the 3-of-5 advanced core electives for EEs and satisfies the 1-of-6 Electrical Engineering Foundations Course requirement for CEs. It is offered in the fall, spring, and summer.